src/main/drivers/light_ws2811strip_hal.c

```diff
@@ -152,12 +160,23 @@ void ws2811LedStripDMAEnable(void)
         return;
     }
 
-    if (HAL_TIM_PWM_Start_DMA(&TimHandle, timerChannel, ledStripDMABuffer, WS2811_DMA_BUFFER_SIZE) != HAL_OK)
+    if(timerNChannel)
     {
-        /* Starting PWM generation Error */
-        ws2811LedDataTransferInProgress = 0;
-        return;
+        if(HAL_TIMEx_PWMN_Start_DMA(&TimHandle, timerChannel, ledStripDMABuffer, WS2811_DMA_BUFFER_SIZE) != HAL_OK)
+        {
```

Comment:
brace style. Can you convert these to the K&R brace style as used throughout betaflight.